scripts/src/main/resources/scripts/command/vscode

```diff
@@ -151,7 +151,7 @@ do
   if [ "${1}" = "run" ] || [ "${1}" = "start" ]
   then
     doSetup silent
-    doStartVsCode
+    doStartVsCode "${@}"
```

Comment:
nice but then you need to call `shift` before as otherwise `run` or `start` will be passed to `doStartVsCode`.

Reply:
Done.